README.md

```diff
@@ -1,2 +1,45 @@
-# Hello-World
-my new repository
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello-World
+
+
+## Built With
+
+- HTML and CSS
+- VSCODE
+- GIT
+
+ 
+
+  
+
+
+
+## Authors
+
+👤 **Alusine Jalloh**
+
+- GitHub: [@githubhandle](https://github.com/Alusp)
+- LinkedIn: [LinkedIn](linkedin.com/in/alusine-jalloh-873951209)
+ 
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly make sure to fix this broken link as it gives an error once clicked to make documentation professional. You can fix it by adding the [MIT.md](https://github.com/microverseinc/readme-template/blob/master/MIT.md) file in your root directory and editing it according to your information.